config/locales/en.yml

```diff
@@ -345,6 +345,10 @@ en:
               too_long: "Reduce the word count for fee details"
             financial_support:
               too_long: "Reduce the word count for financial support"
+        publish/course_location_form:
```

Comment:
Namespacing seems unusual?

Reply:
We need to specify the module path so the translations can be picked up correctly by Rails. It expects this pattern when customising activemodel error messages under module namespaces